python/tests/test_multilevel_atom.py

```diff
@@ -1,13 +1,11 @@
-from __future__ import division
-
+import meep as mp
 import math
 import unittest
 
-import meep as mp
-
 
 class TestMultiLevelAtom(unittest.TestCase):
 
+    @unittest.skipIf(mp.is_single_precision(), "double-precision floating point specific test")
```

Comment:
Would be good to open an issue about this and ping Alex Cerjan.  I'm not sure why a Maxwell–Bloch simulation would be super sensitive to floating-point errors.

Reply:
Maybe we are entering some sort of chaotic regime, in which the precise details (phase etcetera) are extremely sensitive to initial conditions and small calculational errors.